docs/subnets/hello-world-precompile-tutorial.md

```diff
@@ -248,20 +252,15 @@ Clone the repository:
 
 <!-- vale off -->
 
-<Tabs
-defaultvalue="subnet-evm-tab" groupId="subnet-evm-tab"
-values={[
-{label: 'subnet-evm-tab', value: 'subnet-evm-tab'},
-{label: 'precompile-evm-tab', value: 'precompile-evm-tab'},
-]}>
-<TabItem value="subnet-evm-tab" groupId="subnet-evm-tab" >
+<Tabs groupId="evm-tabs">
+<TabItem value="subnet-evm-tab" label="Subnet-EVM" default>
```

Comment:
Do we need `tab` on the name as in `subnet-evm-tab`? It seems redundant.

Reply:
agreed it is redundant. Since it's not showing in front-end, will include in a future PR. We are working on another PR that may replace this page entirely https://github.com/ava-labs/avalanche-docs/pull/1449